website/content/docs/concepts/domain-model/index.mdx

```diff
@@ -51,7 +51,26 @@ The highest-level in Boundary utilizes the concepts of [Roles](/docs/concepts/do
 
 ## Target and Host Resources
 
-Boundary exposes endpoints to users as [Targets](/docs/concepts/domain-model/targets). A target is a resource representing a networked service, with an associated set of permissions, that allows a user to connect and interact with Boundary within a single  session. A target may contain references to [Host](/docs/concepts/domain-model/hosts) sources and [Credential](/docs/concepts/domain-model/credentials) sources. Users who access a target create an authorized session to one of its host sources with returned credentials from the target’s credential source.
+Boundary exposes endpoints to users as [Targets](/docs/concepts/domain-model/targets). A target is a resource representing a networked service, with an associated set of permissions, that allows a user to connect and interact with Boundary within a single session. A target may contain references to [Host](/docs/concepts/domain-model/hosts) sources or an address, and [Credential](/docs/concepts/domain-model/credentials) sources. Users who access a target create an authorized session to its address or one of its host sources with returned credentials from the target’s credential source.
+
+A target exposes two distinct and mutually exclusive configuration paths for its
+representation of a networked service:
+
+- Target with an address
+- Target with host sources
+
+A target with an address represents a target that is set up with a single IP
+address or DNS name directly on the target resource. This mechanism provides
+benefits for new Boundary users and users who don't require Boundary's
+flexibility and scalability, as it's less complex to set up and understand.
+Because of its simplicity, using this path on environments where a significant
+number of resources exist is not recommended.
```

Comment:
```suggestion
Because of its simplicity, using targets with an address is not recommended
when there are a either a large number of hosts and/or dynamic hosts.
```

Reply:
Addressed: https://github.com/hashicorp/boundary/commit/cf145792f9ae1161ddc9521953f39ccabd116bab